src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java

```diff
@@ -155,6 +158,13 @@ private void setUpFiles(Bundle state) {
         setUpFilesState(state);
     }
 
+    //For animation
```

Comment:
This comment can be removed as the method name already indicates that this is being used for setting up an animation.